easylist_adult/adult_specific_hide.txt

```diff
@@ -635,11 +635,13 @@ loverslab.com##.ll_adblock
 vxxx.com##.lmetceehehmmll
 javhd.run##.loading-ad
 hdsex.org##.main-navigation__link--webcams
+adultgamesworld.com##.masonry-layout__panel-content a[href^="https://access-the-website.com/"]
```

Comment:
Maybe fine with `##a[href^="https://access-the-website.com/"]`, there's a link in contents pages too e.g. `https://adultgamesworld.com/2025/02/21/planet-lust-skullgear/`